symphony/lib/toolkit/class.emailgateway.php

```diff
@@ -57,6 +57,7 @@ class EmailValidationException extends EmailGatewayException
     protected $_text_plain;
     protected $_text_html;
     protected $_attachments = array();
+    protected $_ignore_attachment_errors = false;
     protected $_reply_to_name;
```

Comment:
I do not like double negatives. I think it would be better to name it `_validate_attachment_errors` and set it to true.